### DISCUSSION TITLE
Stop processing rewrite rules on match

### DISCUSSION THREAD

**Discussion by michael-letcher** [lwsjs/local-web-server#180] [Q&A]
Is it possible to mimic the "stopProcessing" flag from web.config in IIS. So once a match is found, we don't process any of the other rewrites?

**Comment by 75lb** [comment]
My apologies, this one slipped through the net.. Could you more post detail about the IIS rule you want to mimic? I will need to know specifically what it does and why. A link to documentation would be a good start..
